dojo/tools/nuclei/parser.py

```diff
@@ -0,0 +1,75 @@
+import json
+import hashlib
+from dojo.models import Finding, Endpoint
+
+
+class NucleiParser(object):
+    """
+    A class that can be used to parse the nuclei (https://github.com/projectdiscovery/nuclei) JSON report file
+    """
+
+    # table to match nuclei severity to DefectDojo severity
+    SEVERITY = {
+        "critical": "Critical",
+        "high": "High",
+        "medium": "Medium",
+        "low": "Low",
+        "info": "Info",
```

Comment:
It seems that defect dojo severity = (Nuclei severity).title()
I have the feeling that this mapping is not needed